mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/config/MetricsCollector.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.mantisrx.server.worker.metrics;
+package io.mantisrx.server.master.client.config;
```

Comment:
Same as above. Doesn't belong here IMO.

Reply:
This one is directly referened by WorkerConfiguration which needs to stay inside control-plane-client.